docker.go

```diff
@@ -195,6 +209,9 @@ func commandBuild(build Build) *exec.Cmd {
 	if build.Pull {
 		args = append(args, "--pull=true")
 	}
+	for _, arg := range build.CacheFrom {
+		args = append(args, "--cache-from", arg)
```

Comment:
Does this still succeed in case all the declared images are missing?
I had a similar issue in my current travis setup with bash and used `${CACHED_IMAGE:-false}` as fallback argument to get it to work:
https://gist.github.com/marcbachmann/16574ba8c614bb3b78614a351f324b86#file-travis-docker-scripts-md